packages/connect-form/src/components/connect-form.tsx

```diff
@@ -48,10 +48,18 @@ const descriptionStyles = css({
   marginTop: spacing[2],
 });
 
+const scrollbarWidth = spacing[2];
 const formContentContainerStyles = css({
   padding: spacing[4],
-  overflow: 'scroll',
+  paddingRight: spacing[4] - scrollbarWidth,
+  overflowY: 'scroll',
```

Comment:
Maybe we could leave it at `auto` and avoid overriding the scrollbar size and paddings? This seems like a pretty standard behavior where the scrollbar will appear when needed and move the content to the side a bit when appearing

```suggestion
  overflowY: 'auto',
```